widget/src/pick_list.rs

```diff
@@ -147,7 +163,7 @@ impl<'a, T: 'a, Message, Renderer> Widget<Message, Renderer>
 where
     T: Clone + ToString + Eq + 'static,
     [T]: ToOwned<Owned = Vec<T>>,
-    Message: 'a,
+    Message: 'a + Copy,
```

Comment:
`Copy` is too restrictive bound, a `Clone` should be used instead, otherwise the Message couldn't contain, for example, a `String`